rest_api/test/test_rest_api.py

```diff
@@ -26,11 +27,22 @@
 
 def test_check_single_worker_warning_for_indexing_pipelines(caplog):
     yaml_pipeline_path = Path(__file__).parent.resolve() / "samples" / "test.in-memory-haystack-pipeline.yml"
-    p = _setup_indexing_pipeline(yaml_pipeline_path, None)
+    p, _ = _load_pipeline(yaml_pipeline_path, None)
 
     assert isinstance(p, Pipeline)
     assert "used with 1 worker" in caplog.text
 
+    yaml_pipeline_path = Path(__file__).parent.resolve() / "samples" / "test.in-memory-haystack-pipeline.yml"
+    p, _ = _load_pipeline(yaml_pipeline_path, "BogusPipeline")
```

Comment:
nit: I'd use a more intuitive name to avoid confusion with the other test case, something like 
```python
p, _ = _load_pipeline(yaml_pipeline_path, "ThisPipelineDoesntExist")
```

Reply:
fixed :+1: